src/lib/cache_dir/cache_dir.ml

```diff
@@ -1,10 +1,19 @@
 open Core
+open Async
 
 let autogen_path = Filename.temp_dir_name ^/ "coda_cache_dir"
 
 let manual_install_path = "/var/lib/coda"
 
-let brew_install_path = "/usr/local/var/coda"
+let brew_install_path =
+  match
+    Thread_safe.block_on_async_exn (fun () ->
```

Comment:
I think that this might do enough I/O during startup to break daemonization.

Reply:
Yes, this is why eg the unit tests are failing: 
```
File "src/lib/command_line_tests/command_line_tests.ml", line 63, characters 4-507: The coda daemon works in background mode threw
(monitor.ml.Error
  ("Process.run failed"
    ((prog ../../app/cli/src/coda.exe)
      (args
        (daemon -background -client-port 1337 -config-directory
          /tmp/coda_spun_test.tmp.170fe9))
      (exit_status (Exit_non_zero 1))
      (stdout
        ("Starting background coda daemon. (Log Dir: /tmp/coda_spun_test.tmp.170fe9)"
          ""))
      (stderr
        (monitor.ml.Error
          (Failure
            "Daemon.check_threads: may not be called if any threads have ever been created")
          ("Raised at file \"stdlib.ml\", line 33, characters 17-33"
            "Called from file \"daemon/src/daemon.ml\", line 11, characters 4-107"
            "Called from file \"daemon/src/daemon.ml\", line 69, characters 2-51"
            "Called from file \"src/app/cli/src/coda.ml\", line 167, characters 11-183"
            "Called from file \"src/deferred0.ml\", line 56, characters 64-69"
            "Called from file \"src/job_queue.ml\" (inlined), line 131, characters 2-5"
            "Called from file \"src/job_queue.ml\", line 170, characters 6-47")))))
```

Reply:
hmm yeah thanks for taking a look. That's too bad